CMakeLists.txt

```diff
@@ -306,22 +313,21 @@ if(BUILD_ONNX_PYTHON)
   set_target_properties(onnx_cpp2py_export PROPERTIES SUFFIX ${PY_EXT_SUFFIX})
   set_target_properties(onnx_cpp2py_export
                         PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-  target_include_directories(onnx_cpp2py_export
-                             PRIVATE "${ONNX_ROOT}"
-                                     "${CMAKE_CURRENT_BINARY_DIR}"
-                                     "${PROTOBUF_INCLUDE_DIRS}"
-                                     "${PYTHON_INCLUDE_DIR}")
+  target_include_directories(onnx_cpp2py_export PUBLIC
```

Comment:
technically it's unnecessary to do such special processing for the include paths of this onnx_cpp2py_export target

Reply:
Why do we need `PUBLIC` here?

Reply:
Initially I just wanted to make it consistent with other targets, and since this target is never meant to be a dependency of others, so PUBLIC or PRIVATE shouldn't matter.
But taken a second thought it's probably better to keep it PRIVATE to follow cmake's rule.